lr_face/utils.py

```diff
@@ -185,3 +185,12 @@ def resize_and_normalize(img, target_size):
         img_pixels /= 255
 
         return img_pixels
+
+
+def fix_tensorflow_rtx():
```

Comment:
ik neem deze code voor waar aan

Reply:
Ja dit is een stukje code dat ik in elk project kopieer omdat ik een RTX kaart heb.